pyproject.toml

```diff
@@ -18,11 +21,11 @@ line_length = 80
 python = "^3.11.2"
 astral = "^2.2"
 google-api-python-client = "^1.12.8"
-google-auth-httplib2 = "^0.0.4"
 google-auth-oauthlib = "^0.4.2"
 pydantic = "^1.7.3"
 click = "^8.0.1"
-rsa = "*"
+google = "^3.0.0"
```

Comment:
So I'm assuming that `google` replaces the `google-auth-httplib2` package?

Reply:
Not exactly. Apparently, according to deptry, google-auth-httplib2 is a dependency of a dependency that we don't need to explicitly list here. And google is a dependency we should list, because right now we are just lucky that it gets pulled in indirectly.

Reply:
the list of imports I was using before was from the official documentation of the google API :-) So I guess times just changed since then.